web/packages/teleport/src/AppLauncher/AppLauncher.tsx

```diff
@@ -72,7 +72,11 @@ export function AppLauncher() {
       window.location.replace(`https://${fqdn}${port}${path}`);
     } catch (err) {
       let statusText = 'Something went wrong';
-      if (err instanceof Error) {
+
+      if (err instanceof TypeError) {
```

Comment:
I'm no expert but `TypeError` seems broad. Would be it better to check if the error text is "Failed to fetch" like what we see is being returned in this case?

@ryanclark @kimlisa I'll defer to you on this one.

Reply:
or we could just add the real error message `err.message` along with the status text